gearmand/datadog_checks/gearmand/gearmand.py

```diff
@@ -115,6 +115,17 @@ def check(self, instance):
                 message="Connection to %s:%s succeeded." % (host, port),
                 tags=tags,
             )
+            self._collect_metadata(client)
         except Exception as e:
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, message=str(e), tags=tags)
             raise
+
+    def _collect_metadata(self, client):
+        resp = client.get_version()
+        if not resp.startswith('OK'):
+            self.log.error('Error retrieving version information from server, response: %s', resp)
```

Comment:
```suggestion
            self.log.info('Error retrieving version information from server, response: %s', resp)
```
I _think_ version failure should just be a warn or info